lib/cadet/assessments/assessment.ex

```diff
@@ -40,7 +41,7 @@ defmodule Cadet.Assessments.Assessment do
     timestamps()
   end
 
-  @required_fields ~w(title open_at close_at number course_id config_id)a
+  @required_fields ~w(title open_at close_at number course_id config_id max_team_size)a
```

Comment:
Can we add a

```elixir
|> validate_number(
  :max_team_size,
  greater_than_or_equal_to: 1
)
```

as well into the changeset?

Reply:
Okie. I will ass this into the changeset!